src/neo4j/time/__init__.py

```diff
@@ -2019,6 +2019,12 @@ def __new__(
         return cls.combine(Date(year, month, day),
                            Time(hour, minute, second, nanosecond, tzinfo))
 
+    # SERIALIZATION #
+
+    def __reduce__(cls):
```

Comment:
This will not work for instances with dynamic attributes (populated `__dict__`).